config/game_options.txt

```diff
@@ -270,6 +270,9 @@ ALLOW_AI
 
 ## AWAY MISSIONS ###
 
+# Will random away mission be loaded at initialization.
```

Comment:
do what now

Reply:
What's the problem ?

Reply:
should have a ? at the end, with the period it makes it read like a statement